webapp/wmservices/src/main/java/org/apache/oodt/cas/wmservices/resources/WorkflowInstanceResource.java

```diff
@@ -56,10 +56,12 @@ public WorkflowInstanceResource(WorkflowInstance workflowInstance) {
     this.workflowInstanceId = workflowInstance.getId();
     this.currentTaskId = workflowInstance.getCurrentTaskId();
     this.startDate = workflowInstance.getStartDate().toString();
-    this.endDate = workflowInstance.getEndDate().toString();
     this.timesBlocked = workflowInstance.getTimesBlocked();
     this.sharedContext = new MetadataResource(workflowInstance.getSharedContext());
     this.workflowState = new WorkflowStateResource(workflowInstance.getState());
+    if (workflowInstance.getEndDate() != null){
```

Comment:
Shall we use StringUtils.isEmpty() for null check

Reply:
But the `getEndDate()` returns a `Date`, not a string. As I found out `!= null` is the correct way to check if a `Date` is null, isn't it?